src/functionalTest/resources/features/F-001/F-001.feature

```diff
@@ -0,0 +1,137 @@
+    Then  a negative response is received
+    And   the response [contains an HTTP 401 'Unauthorized']
+    And   the response has all other details as expected
+
+  @S-006
+  Scenario: must return a negative response 403 when request provides authentic credentials without authorised access
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permissions on a document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   the request [user provides a wrong service Authorisation token]
+    And   it is submitted to call the [Retrieval of metadata from doc-store for given document id] operation of [CCD data store]
+    Then  a negative response is received
+    And   the response [contains an HTTP 403 'Forbidden']
+    And   the response has all other details as expected
+
+  @S-007
+  Scenario: Receive an error response, for retrieval of doc-store metadata for non-existing document id
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permissions on a document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   the request [for retrieval of retrieval doc-store metadata for non-existing document id]
+    And   it is submitted to call the [Retrieval of metadata from doc-store for given document id] operation of [CCD data store]
+    Then  a negative response is received
+    And   the response [contains an HTTP 404 'Not Found']
+    And   the response has all the details as expected
+
+  @S-008
+  Scenario: Receive an error response, for retrieval of doc-store metadata for document id which is not associated
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permissions on a document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    And   a user with [an active citizen profile in CCD with full permission on specific document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   the request [for retrieval of doc-store metadata for document id uploaded by citizen]
+    And   it is submitted to call the [Retrieval of metadata from doc-store for given document id] operation of [CCD data store]
+    Then  a negative response is received
+    And   the response [contains an HTTP 403 'Forbidden']
```

Comment:
Let's not mention the http codes in scenarios any more - they should not be specified by BAs, but should follow after design of the API and appear in the test data as per the design.